include/worker.h

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright © 2017 Drew Walters <drewwalters96@gmail.com>
+ * SPDX-License-Identifier: (BSD-3-Clause OR GPL-2.0)
+ */
+
+#ifndef WORKER_H
+#define WORKER_H
+
+/* Queue worker function capacity. */
+#define MAX_WORKERS 1
+
+/*
+ * Type definition of worker function added to queue by interrupt handler.
+ */
+typedef int (*worker)(void *);
+
+/*
+ * Queues a worker function when called by an interrupt handler.
+ */
+void queue_worker(worker);
```

Comment:
Please add the parameter name here. Also, do you have any opinion on worker vs. work_function as the typedef name?

Reply:
Done. I do not, so I have changed it to `work_function`.